src/UnifiedFFI/FFILibrary.class.st

```diff
@@ -53,17 +53,19 @@ FFILibrary >> calloutAPIClass [
 
 { #category : #'library path' }
 FFILibrary >> ffiLibraryName [
-	^ self moduleName
+	^ self libraryName
 ]
 
-{ #category : #'accessing platform' }
-FFILibrary >> macModuleName [ 
```

Comment:
Is it possible to deprecate the method instead of removing it to ease migration of existing FFI bindings?

Reply:
This method is meant to be overridden. Users overriding it will still work as it is still called :) I did it in such a way that it is backwards compatibility.